packages/tradershub/src/features/cfd/flows/OtherCFDs/Dxtrade/AvailableDxtradeAccountsList/AvailableDxtradeAccountsList.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { Fragment, useState } from 'react';
 import { useRegulationFlags } from '@/hooks';
+import { useActiveTradingAccount } from '@deriv/api';
```

Comment:
same here

Reply:
As per discussed with @matin-deriv we will leave the naming as it is for now.